src/app/api/places/chat/route.js

```diff
@@ -0,0 +1,121 @@
+import { generateText } from "ai";
+import { createOllama } from "ollama-ai-provider";
+
+const generateGoogleFindUrlUsingLLM = async (userLocationString, messages) => {
+  const llama = createOllama();
+  return generateText({
+    model: llama("openhermes2.5-mistral"),
+    system: `Your only task is to construct a valid Google Find Place API URL. 
+    You are allowed to use information provided in system prompt and the conversation.
+
+    Additional information:
+    ${userLocationString}
+    
+    Additional guidelines you should follow:
+      - The URL should be wrapped between three backticks
+      - Use radius of 2000 if it can't be determined from the conversation
```

Comment:
unit is missing, 2000 meters I assume?

Reply:
The LLM will understand as docs talk about radius in numbers